vkconfig_core/environment.cpp

```diff
@@ -319,6 +319,25 @@ static QString GetDefaultExecutablePath(const QString& executable_name) {
 #error "Unknown platform"
 #endif
 
+#if PLATFORM_MACOS
```

Comment:
Please use if (PLATFORM_MACOS) so that it always get compile on all platforms

Reply:
Okay, done here.